File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/PacijentiResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class PacijentiResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int PacijentID { get; internal set; }
            public string Ime { get; set; }
            public string Prezime { get; set; }
            public DateTime DatumRodjenja { get; set; }
            public string Telefon { get; set; }
            public string Adresa { get; set; }
            public string Email { get; set; }

        }
    }
}

File: app/src/main/java/com/example/ordinacija/Data/PacijentiResultVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class PacijentiResultVM implements Serializable {



    public static class Row implements Serializable {
        public int PacijentID;
        public String Ime;
        public String Prezime;
        public Date DatumRodjenja;
        public String Telefon;
        public String Adresa;
        public String Email;
        public  Row(){

        }

    }
    public List<Row> rows;
    public PacijentiResultVM() {
    }

}
